GUI/src/types/chat.ts

```diff
@@ -24,6 +24,8 @@ export enum CHAT_EVENTS {
   ASK_PERMISSION_ACCEPTED = 'ask-permission-accepted',
   ASK_PERMISSION_REJECTED = 'ask-permission-rejected',
   ASK_PERMISSION_IGNORED = 'ask-permission-ignored',
+  ASK_TO_FORWARD_TO_CSA = 'ask_to_forward_to_csa',
```

Comment:
I do not think I get this. You use snake case here (`ask_to_forward_to_csa`) but kebab case in translation JSON files (`ask-to-forward-to-csa`). This will cause issues, right?